src/events/pullRequest/index.ts

```diff
@@ -18,47 +18,57 @@ export async function handlePullRequestEvent(
   contributorService: IContributorService
 ) {
   const { action, pull_request: pullRequest } = context.payload;
+  const pullKey = context.pullRequest();
 
-  switch (action) {
-    case "opened":
-    case "edited":
-    case "reopened":
-    case "labeled":
-    case "unlabeled":
-    case "synchronize":
-      // Synchronize pull request.
-      await pullService.syncPullRequest({
-        ...context.repo(),
-        ...pullRequest,
-      });
-      break;
-    case "closed":
-      // Synchronize pull request.
-      await pullService.syncPullRequest({
-        ...context.repo(),
-        ...pullRequest,
-      });
+  // Sync pull request.
+  if (
+    action === "opened" ||
```

Comment:
Maybe you can add an enumeration to it.

Reply:
I've thought about it. But in fact, enum will not bring us any benefits, because the type definition of `action` has restricted the allowed values.

Also, I want to fall back to using the switch statement.

Reply:
Got it.